tests/integration/ceph_helm_test.go

```diff
@@ -46,7 +46,7 @@ var (
 // - Create the object store via the CRD
 // ***************************************************
 func TestCephHelmSuite(t *testing.T) {
-	if installer.SkipTestSuite(installer.CephTestSuite) {
+	if installer.SkipTestSuite(installer.CephTestSuite) || RookTestOpenshift {
```

Comment:
This is another place where a separate environment variable would be preferred instead of `RookTestOpenshift`. It could be something like this:
```
if installer.SkipTestSuite(installer.CephTestSuite) || os.Getenv("ROOK_SKIP_CEPH_HELM_SUITE") == "true" {
  t.Skip()
}
```

Similarly for other tests that are being skipped for openshift. This flag could also be used in the future for some other test environment besides openshift.